### DISCUSSION TITLE
When the CI fails due to a missing dep, the action doesn't actually fail.

### DISCUSSION THREAD

**Discussion by adambirds** [squidfunk/mkdocs-material#4758] [Q&A] [answered]
Thought I'd ask about it here first as not sure if it is an issue with the theme or an issue mkdocs itself.

You can see in this pipeline under the deploy section.

https://github.com/adb-web-designs/vanderwood-twitch-bot/actions/runs/3688574185/jobs/6243528700

Should nthe issue be raised her or with mkdocs itself?

**Comment by squidfunk** [comment, marked as answer]
This was an error of the social plugin, that did not set the correct error code when exiting. [It's already fixed in v9](https://github.com/squidfunk/mkdocs-material/blob/feature/material-v9/src/plugins/social/plugin.py#L83).

**Reply by adambirds** [reply]
Thanks
